templates/lxd/index.html

```diff
@@ -4,337 +4,428 @@
+            </p>
+          </div>
         </div>
       </div>
     </div>
-  </div>
-</section>
+  </section>
 
-<section class="p-section">
-  <div class="row--50-50">
-    <hr class="p-rule">
-    <div class="col">
-      <h3 class="p-heading--2">Learn more about LXD</h3>
-    </div>
-    <div class="col">
-      <div class="row--25-75">
-        <div class="col">
-          <h3 class="p-text--small-caps">Webinars</h3>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h3 class="p-heading--2">Learn more about LXD</h3>
+      </div>
+      <div class="col">
+        <div class="row--25-75">
+          <div class="col">
+            <h3 class="p-text--small-caps">Blogs</h3>
+          </div>
+          <div class="col">
+            <p>
+              <a href="https://ubuntu.com/blog/lxd_5-21-0_lts">LXD 5.21.0 LTS is now available&nbsp;&rsaquo;</a>
+            </p>
+            <p>
+              <a href="https://ubuntu.com/blog/esxi-alternative">ESXi Alternative: try open source LXD&nbsp;&rsaquo;</a>
+            </p>
+            <p>
+              <a href="https://ubuntu.com/blog/using-lxd-to-host-your-own-erp-cost-effectively">Using LXD to host your own ERP, cost-effectively&nbsp;&rsaquo;</a>
+            </p>
+            <p>
+              <a href="https://ubuntu.com/blog/open-source-for-beginners-dev-environment-with-lxd">Open source for beginners: setting up your dev environment with LXD&nbsp;&rsaquo;</a>
+            </p>
+          </div>
         </div>
-        <div class="col">
-          <p><a href="https://www.brighttalk.com/webcast/6793/298235">Learn the basics of LXD&nbsp;&rsaquo;</a></p>
-          <p>
-            <a
-              href="https://www.brighttalk.com/webcast/6793/470152?utm_source=brighttalk-portal&utm_medium=web&utm_campaign=viewing-history">Run
+        <hr class="p-rule" />
+        <div class="row--25-75">
+          <div class="col">
+            <h3 class="p-text--small-caps">Webinars</h3>
+          </div>
+          <div class="col">
+            <p>
+              <a href="https://www.brighttalk.com/webcast/6793/298235">Learn the basics of LXD&nbsp;&rsaquo;</a>
+            </p>
+            <p>
+              <a href="https://www.brighttalk.com/webcast/6793/470152?utm_source=brighttalk-portal&utm_medium=web&utm_campaign=viewing-history">Run
               Kubernetes on LXD for edge computing&nbsp;&rsaquo;</a>
-          </p>
-        </div>
-      </div>
-      <hr class="p-rule">
-      <div class="row--25-75">
-        <div class="col">
-          <h3 class="p-text--small-caps">Whitepaper</h3>
+            </p>
+          </div>
         </div>
-        <div class="col">
-          <p>
-            <a href="https://ubuntu.com/engage/lxd-whitepaper">How to build a lightweight system container
+        <hr class="p-rule" />
+        <div class="row--25-75">
```

Comment:
For now row--25-75 and row--50-50 are not nested in each other, I understand it would have been better to get all of them in `.row ` format but it wasn't possible by design.